src/main/java/frc/robot/commands/SetHoodCommand.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.ShooterHoodSubsystem;
+
+/**
+ * SetHoodCommand
+ */
+public class SetHoodCommand extends CommandBase{
+
+  private final ShooterHoodSubsystem shooterHoodSubsystem;
+  private final double percentageExtended;
+
+  public SetHoodCommand(ShooterHoodSubsystem shooterHoodSubsystem, double percentageExtended) {
+    this.shooterHoodSubsystem = shooterHoodSubsystem;
+    this.percentageExtended = percentageExtended;
```

Comment:
The command should require the subsystem.
```suggestion
    this.percentageExtended = percentageExtended;
    addRequirements(shooterHoodSubsystem);
```

Reply:
Just realized you added a suggestion, I already added the same.